### PR TITLE
[ADD] partner_no_unique_bank

### LINKED ISSUE #241: Remove disabled constraint unique_number on res.partner.bank
OCA won't accept this disabled constraint. We should move it to a custom module or sanitise our clients' databases.

https://gestion.coopiteasy.be/web#id=9262&action=475&active_id=221&model=project.task&view_type=form&menu_id=536

Fixed by https://github.com/coopiteasy/vertical-cooperative/pull/310

Replaced by https://github.com/coopiteasy/addons/pull/226

- [x] deploy in test
- [x] notify Pol
- [ ] merge 
- [ ] and port to 14.0

### PR DESCRIPTION
**Pull request by carmenbianca**
## Description

Related to https://github.com/coopiteasy/vertical-cooperative/pull/310, fixes https://github.com/coopiteasy/vertical-cooperative/issues/241

## Odoo task (if applicable)

Related task: https://gestion.coopiteasy.be/web#id=8660&action=479&model=project.task&view_type=form&menu_id=536

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Review by robinkeunen — APPROVED** [review]
finally 😅

File: partner_no_unique_bank/models/res_partner_bank.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+from odoo import models
+
+
+class ResPartnerBank(models.Model):
+    _inherit = "res.partner.bank"
+
+    _sql_constraints = [
+        ("unique_number", "Check(1=1)", "Account Number must be unique!")
```
Comment (enricostano) [inline partner_no_unique_bank/models/res_partner_bank.py:13]: Hi! We ended up doing something similar... but taking in account companies:

https://git.coopdevs.org/coopdevs/odoo/odoo-addons/odoo-addons/-/blob/master/remove_dup_acc_num_constraint/models/res_partner_bank.py#L7

**Pull request merged**